client/src/sections/resources/PowerfulGlove.tsx

```diff
@@ -27,7 +27,8 @@ const PowerfulGlove = () => {
     <Tile
       header="Powerful Glove"
       imageUrl="/images/itemimages/Pglove.gif"
-      itemToEquip={$item`Powerful Glove`}
+      linkedContent={$item`Powerful Glove`}
+      linkHide={useHaveEquipped($item`Powerful Glove`)}
```

Comment:
do we want to make linkHide something that automatically generates via the way you constructed DynamicLink? seems a little unnecessary to have to port $item`` into two different Tile props in addition to the hide= prop. think we can probably check equip status within DynamicLink and have that pass linkHide itself.

Reply:
agree

Reply:
I would start by getting just making DynamicLink hide itself if you don't `have` the relevant thing.

Reply:
Don't